examples/test/post-processing/components/bloom.js

```diff
@@ -0,0 +1,111 @@
+/* global AFRAME THREE */
+
+/*
+Kawase bloom, described here
+https://software.intel.com/en-us/blogs/2014/07/15/an-investigation-of-fast-real-time-gpu-based-image-blur-algorithms
+*/
+
+AFRAME.registerComponent('bloom', {
```

Comment:
We could but it would be limiting. A postproc effect usually employs several shaders and passes to produce. For instance an ssao pass first runs the ssao shader which gets out noisy so it also applies a special blur to it, taking depth into account so as to avoid spilling the ao to near objects. The components are the best solution imo, and with the graph mechanism could be used in creative ways, like defining entities that each contain a whole chain of postprocessing filters and combine these entities